_docs/tasks/istio-auth.md

```diff
@@ -1,37 +1,153 @@
 ---
 title: Enabling Istio Auth
 overview: This task shows you how to setup Istio-Auth to provide mutual TLS authentication between services.
-  
+
 order: 70
 
 bodyclass: docs
 layout: docs
 type: markdown
 ---
 
-This task shows how to do X in a Kubernetes cluster. You'll learn
-how to ...
+This task shows how to set up Istio auth in a Kubernetes cluster. You'll learn
+how to:
+* Enable Istio auth
+* Disable Istio auth
+* Verify Istio auth setups
 
 
 ## Before you begin
-* Do this.
-* Do this too.
+The tutorial assumes you have:
+* Read the [Istio auth
```

Comment:
Here and else, please add a blank line between bullet items for better HTML rendering.